packages/thirdweb/src/react/core/utils/wallet.ts

```diff
@@ -110,11 +117,31 @@
   });
 
   const addressOrENS = ensNameQuery.data || shortAddress;
+  const pfpUnresolved = socialProfileQuery.data?.filter((p) => p.avatar)[0]
```

Comment:
Which did you set as the first in the backend I'm the end?

Reply:
ENS, Farcaster, then Lens